src/sc_generation/template/wasmv1/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "test",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "asc src/$SC_NAME.ts --use abort=src/$SC_NAME/myabort --target release --outFile build/$SC_NAME.wasm && wasmv1_transform build/$SC_NAME.wasm add",
+    "build_windows": "asc src/%SC_NAME%.ts --use abort=src/%SC_NAME%/myabort --target release --outFile build/%SC_NAME%.wasm && wasmv1_transform build/%SC_NAME%.wasm add",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "assemblyscript": "^0.26",
+    "typescript": "^4.8.4"
+  },
+  "dependencies": {
+    "as-proto": "^1.3.0",
+    "massa-proto-as": "/home/jf/workspace/rust/massa-abi-metaproject/massa-proto-as"
```

Comment:
Hardcoded path?

Reply:
:eyes: I don't know

yes we need to fix this before merge. 
If I'm correct, it implies that `massa-proto` is merged first, so massa-proto-as is updated...

Reply:
@modship 
if it's ok for you, I leave this point open, and you close it when you correct this line with the merged massa-proto-as?

Reply:
i updated this part to target massa-proto-as on deferred_calls branch for now